bodhi/client/__init__.py

```diff
@@ -84,6 +103,23 @@ def new(user, password, **kwargs):
 
 
 @updates.command()
+@add_options(new_edit_options)
+@click.argument('builds')
```

Comment:
Ok, I believe this will need some changes on the server side too. I have to look at it.